src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -62,10 +62,10 @@ public void setTestVerbosity(){
     public static final String publicTestDir = new File(gatkDirectory, publicTestDirRelative).getAbsolutePath() + "/";
     public static final String publicTestDirRoot = publicTestDir.replace(publicTestDirRelative, "");
 
-    public static final String GSC_GATK_TEST_RESOURCES = "gs://hellbender/test/resources/";
+    public static final String GCS_GATK_TEST_RESOURCES = "gs://hellbender/test/resources/";
```

Comment:
Oh, yeah, we have an environment variable for this so people can redirect to their own copy of the test resources if they want.  HELLBENDER_TEST_INPUTS.  We should use that instead.